### DISCUSSION TITLE
Failed to detect the language of repo written in Maxima

### DISCUSSION THREAD

**Discussion by cosargozukirmizi** [github-linguist/linguist#5789] [Classification] [answered]
Maxima is the language for Maxima Computer Algebra System.

**Comment by lildude** [comment, marked as answer]
Linguist doesn't know about Maxima so this really should have been a [New language](https://github.com/github/linguist/issues/new?assignees=&labels=Add+Language%2C+Good+First+Issue&template=new_language.md&title=) issue.

An attempt to add Maxima was made a few years ago in https://github.com/github/linguist/pull/4775 but it went stale.

If you've got the bandwidth, feel free to create a new PR to add support, else please open a new issue using the link above or the "New language" option if you open an issue directly in the repo.

**Comment by Observatorio-de-Matematica** [comment]
Recently, agent mode has been made available with GitHub Copilot in VS Code. So, one can try use it to manage the process for a pulll request aiming for a language to be recognized.
